lib/app_state/synchronization_state/synchronization_state.dart

```diff
@@ -13,27 +13,35 @@ class SynchronizationState with ChangeNotifier {
   bool _isDataDownloadingActive;
   bool _hasUnsyncedData;
   bool _isUnsyncedCheckingActive = true;
+  bool _isCheckingForAvailableDataFromServer;
   SynchronizationService _synchronizationService;
   int _beneficiaryCount;
   int _beneficiaryServiceCount;
+  int _conflictLCount = 0;
+  String _statusMessageForAvailableDataFromServer;
   List<String> _dataDownloadProcess;
   List<String> _dataUploadProcess;
   List<Events> _eventFromServer;
   List<TrackeEntityInstance> _trackeEntityInstance;
   List<dynamic> _servertrackedEntityInstance;
+  List<Map<String, dynamic>> _events_1;
+  List<Map<String, dynamic>> _trackedInstance1;
```

Comment:
Why _trackedInstance1 ??

Reply:
I realy dont know, I found that part of code in use and i dont know a thing about it